### PR TITLE
增加aoeiuv的声调的unicode字符的支持（非数字)

### PR DESCRIPTION
**Pull request by cuihq**
使用中遇到了同样的问题( #9 #14 ) ,然后修改了一下,请审查并修改后合并到新版本.:smile:

```ruby
Pinyin.t('中国', tonemarks: true) => "zhōng guó"
```

### CODE REVIEW

File: chinese_pinyin.gemspec [hunk 1]
```diff
@@ -7,8 +7,8 @@ Gem::Specification.new do |s|
   s.name        = "chinese_pinyin"
   s.version     = ChinesePinyin::VERSION
   s.platform    = Gem::Platform::RUBY
-  s.authors     = ["Richard Huang", "Hong, Liang"]
-  s.email       = ["flyerhzm@gmail.com", "hongliang@bamajia.com"]
+  s.authors     = ["Richard Huang", "Hong, Liang", "cuihq"]
+  s.email       = ["flyerhzm@gmail.com", "hongliang@bamajia.com", "cuihaiqin@gmail.com"]
```
Comment (flyerhzm) [inline chinese_pinyin.gemspec]: sorry only who contributes many times can be added to authors

File: lib/chinese_pinyin/version.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # encoding: utf-8
 module ChinesePinyin
-  VERSION = "0.6.1.1"
+  VERSION = "0.6.2.0"
```
Comment (flyerhzm) [inline lib/chinese_pinyin/version.rb]: you don't need to worry about version

File: lib/chinese_pinyin.rb [hunk 3]
```diff
@@ -41,7 +51,8 @@ def init_word_table
 
     def translate(chars, options={})
       splitter = options.fetch(:splitter, ' ')
-      tone     = options.fetch(:tone, false)
+      tonemarks= options.fetch(:tonemarks, false)
```
Comment (flyerhzm) [inline lib/chinese_pinyin.rb]: please add a space between tonemarks and =

**Comment by cuihq** [comment]
修改了代码风格，还原了版本号及版权声明,请复查并修改后合并到新版本.:smile:

File: chinese_pinyin.gemspec (hunk 1 above)
Comment (cuihq) [inline chinese_pinyin.gemspec]: I see.

File: lib/chinese_pinyin/version.rb (hunk 2 above)
Comment (cuihq) [inline lib/chinese_pinyin/version.rb]: as above

File: lib/chinese_pinyin.rb (hunk 3 above)
Comment (cuihq) [inline lib/chinese_pinyin.rb]: :ok_hand:

**Pull request merged**